solc/CommandLineParser.cpp

```diff
@@ -509,9 +511,11 @@ void CommandLineParser::parseOutputSelection()
 			"The following outputs are not supported in " + g_inputModeName.at(m_options.input.mode) + " mode: " +
 			joinOptionNames(unsupportedOutputs) + "."
 		);
+
+	// TODO: restrict EOF version to correct EVM version.
```

Comment:
When TODO?

Reply:
Once the EVM version that will have EOF is actually known.